evap/student/templates/student_vote.html

```diff
@@ -154,7 +154,7 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
                             <span class="fas fa-circle-info" data-bs-toggle="tooltip" title="{% trans 'The evaluation can be continued later using the same device and the same browser. But you have to submit it to send it to the server and make it count. After submitting, you can not edit the evaluation anymore.' %}"></span>
                         </div>
                         <button id="vote-submit-btn" type="submit" class="btn btn-success tab-selectable">{% trans 'Submit questionnaire' %}</button>
-                        <p id="submit-error-warning" style="display: none">
+                        <p id="submit-error-warning" style="display: none", class="text-danger">
```

Comment:
HTML attributes are typically separated just by whitespace. I don't know if the comma is legal here, but for consistency in the code base, I'd rather have tno comma here.